test/e2e/devsandbox-dashboard/README.md

```
@@ -0,0 +1,45 @@
+
+*Prerequisites*:
+
+1. You need a OCP cluster
+    - ROSA cluster from ClusterBot will not work since we are not able to modify the OAuth configuration of ROSA clusters created by the ClusterBot.
+2. Ensure you are using Node.js version 22
+    - to easily manage it, you can run `nvm use 22`
+3. Ensure you have `yarn` installed
+4. Make sure you can log in at <https://sso.devsandbox.dev/auth/realms/sandbox-dev/account> using your SSO_USERNAME and SSO_PASSWORD
+5. Make sure you have toolchain resources deployed on your cluster (you can run `make prepare-and-deploy-e2e`)
+
+### Running UI E2E Tests locally
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)